bazel/rules.bzl

```diff
@@ -38,4 +35,6 @@ def cris_cc_binary(**attrs):
     native.cc_binary(**__add_opts(attrs))
 
 def cris_cc_test(**attrs):
+    if "linkstatic" not in attrs:
+        attrs["linkstatic"] = False
```

Comment:
- Do we need to specify anything on lib side? 
- Not every lib should be linked as shared, especially protobuf.

Reply:
> - Do we need to specify anything on lib side? 

not for now 

> - Not every lib should be linked as shared, especially protobuf.

That is exactly why not